jupyterhub/scopes.py

```diff
@@ -64,7 +64,7 @@
         'subscopes': ['read:users:name'],
     },
     'read:users': {
-        'description': 'Read user models (including servers, tokens and authentication state).',
+        'description': 'Read user models (including state of the default server, tokens and authentication state).',
```

Comment:
Thanks for noticing! Neither tokens nor auth state are covered by read:users either, so this somehow got quite inaccurate. It is also not the _state_ of the default server (server_state is a thing covered by its own scope), but rather its URL if it's running. How about:

```suggestion
        'description': 'Read user models (including the URL of the default server if it is running).',
```